.github/workflows/build.yml

```diff
@@ -23,25 +23,32 @@ jobs:
         run: env | sort
 
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v4
         with:
           fetch-depth: 1
 
       - name: Setup Graalvm
         uses: graalvm/setup-graalvm@v1
         with:
-          java-version: '17.0.8'
+          java-version: '17'
           distribution: 'graalvm'
           github-token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Print info
+        run: |
+          echo "GRAALVM_HOME: $GRAALVM_HOME"
+          echo "JAVA_HOME: $JAVA_HOME"
+          java --version
+          native-image --version
+
       - name: Install native-image component
         run: gu install native-image
 
       - name: Tests
-        run: ./gradlew check
+        run: ./gradlew test
```

Comment:
I'm just testing things, since in Mac when we run the tests it works, but the native ones fail with this [error](https://github.com/oracle/graal/issues/478)...

Is it correct that it should always run the tests with the native image? Why are there 2 different folders if they are both run with the native image?

Reply:
AFAIK tests run both using classic JVM and as native image. Which 2 folders are you referring ?